src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -923,16 +928,30 @@ public class Color : Disposable, ICloneable
             // Do nothing. Just call for load static values.
         }
 
+        internal static System.Runtime.InteropServices.HandleRef getCPtr(Color obj)
+        {
+            return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.SwigCPtr;
+        }
+
+        internal new System.Runtime.InteropServices.HandleRef SwigCPtr
+        {
+            get => CopyValueAndReturnSingletonHandleRef();
+        }
+
+        private System.Runtime.InteropServices.HandleRef CopyValueAndReturnSingletonHandleRef()
+        {
+            Vector4.SingletonDaliVector4.SetValues(r, g, b, a);
+            return Vector4.SingletonDaliVector4.SwigCPtr;
+        }
+
         /// <summary>
         /// Default constructor
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
-        public Color() : this(Interop.Vector4.NewVector4(), true)
+        public Color()
```

Comment:
```suggestion
        public Color() : this(0.0f, 0.0f, 0.0f, 0.0f)
```

Don't we need to initialize the local variables?

Reply:
This patch is using to remove the DALi instance from Color, so it don't need to do this.